core/src/main/java/org/jdbi/v3/core/mapper/InternetMapperFactory.java

```diff
@@ -67,7 +67,7 @@ private static InetAddress getInetAddress(ResultSet r, int i, StatementContext c
         try {
             return hostname == null ? null : InetAddress.getByName(hostname);
         } catch (UnknownHostException e) {
-            throw new MappingException("Could not map InetAddress", e);
+            throw new MappingException("Could not map hostname " + hostname + " to InetAddress", e);
```

Comment:
this is not needed, as the UnknownHostException is constructed as `new UnknownHostException(hostname)` :-)

Reply:
true :)